release: 1.3.0

ematipico opened the pull request:
<!--
	Thanks for submitting a Pull Request! We appreciate you spending the time to work on these changes.
	Please provide enough information so that others can review your PR.
	Once created, your PR will be automatically labeled according to changed files.
	Learn more about contributing: https://github.com/biomejs/biome/blob/main/CONTRIBUTING.md
-->

## Summary

Release 1.3.0

<!-- Explain the **motivation** for making this change. What existing problem does the pull request solve?-->

<!-- Link any relevant issues if necessary or include a transcript of any Discord discussion. -->

## Test Plan

Check documentation, changelog and new versions of the crates.

<!-- What demonstrates that your implementation is correct? -->

SuperchupuDev commented on website/src/content/docs/reference/configuration.mdx:
```diff
@@ -744,3 +742,61 @@ How big the indentation should be for JSON (and its super languages) files.
+
+## `overrides`
+
+A list of pattern .
```
```suggestion
A list of patterns.
```

SuperchupuDev commented on website/src/content/docs/reference/configuration.mdx:
```diff
@@ -744,3 +742,61 @@ How big the indentation should be for JSON (and its super languages) files.
+
+### `overrides.<ITEM>.linter`
+
+It will include the options of [top level formatter](#linter), minus `ignore` and `include`.
```
you forgot to change this (and the ones below it) 👀

ematipico commented:
https://biome-website-fbe7abdn7-ematipico.vercel.app/reference/configuration/#overrides

The preview. There's still one typo, but it should be fine as a start

ematipico commented:
Last changes:
- overrides weren't working for the parser options, now they are
- updated the `usage` heading with more info (basic) about the main commands https://biome-website-eqyb49zk8-ematipico.vercel.app/guides/getting-started/#usage
- added examples in the overrides section https://biome-website-eqyb49zk8-ematipico.vercel.app/reference/configuration/#overrides
